mindmeld/components/question_answerer.py

```diff
@@ -833,7 +833,14 @@ def execute(self, size=10):
             es_query = self._build_es_query(size=size)
 
             response = self.client.search(index=self.index, body=es_query)
-            results = [hit["_source"] for hit in response["hits"]["hits"]]
+
+            # construct results, removing embedding metadata and exposing score
+            results = []
+            for hit in response["hits"]["hits"]:
+                item = {key: val for (key, val) in hit["_source"].items()
+                        if not key.endswith(EMBEDDING_FIELD_STRING)}
+                item['_score'] = hit['_score']
```

Comment:
Small nit: should we use `score` instead of `_score` since "_" tends to denote private fields and properties?

```suggestion
                item['_score'] = hit['_score']
```

Reply:
I wanted it to include some special character to it's less likely to overwrite any fields of the item itself. E.g. if there was an index of soccer games there might be a `score` field already